src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -810,20 +803,17 @@ private async Task<ContentValidateResult> UpdateContentItemVersionAsync(ContentI
 
                 await Handlers.InvokeAsync((handler, context) => handler.PublishingAsync(context), publishContext, _logger);
                 await ReversedHandlers.InvokeAsync((handler, context) => handler.PublishedAsync(context), publishContext, _logger);
-
-                // Restore values that may have been altered by handlers.
-                if (modifiedUtc.HasValue)
-                {
-                    updatedVersion.ModifiedUtc = modifiedUtc;
-                }
-                if (publishedUtc.HasValue)
-                {
-                    updatedVersion.PublishedUtc = publishedUtc;
-                }
             }
 
-            // Re-enlist content item here in case of session queries.
-            _session.Save(updatingVersion);
+            // Restore values that may have been altered by handlers.
+            if (modifiedUtc.HasValue)
+            {
+                updatingVersion.ModifiedUtc = modifiedUtc;
+            }
+            if (publishedUtc.HasValue)
+            {
+                updatingVersion.PublishedUtc = publishedUtc;
+            }
```

Comment:
@deanmarcussen 

Can you confirm that this was not at the right place and that it was wrong to use `updatedVersion`, replaced here by `updatingVersion`

Reply:
@jtkech yes I think so, bad cut and paste originally :(